bciers/apps/reporting/src/app/components/operations/OperationReview.tsx

```diff
@@ -0,0 +1,161 @@
+"use client";
+import {
+  Box,
+  Typography,
+  Button,
+  Paper,
+  Grid,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+} from "@mui/material";
+import InfoIcon from "@mui/icons-material/Info";
+import SyncIcon from "@mui/icons-material/Sync";
+import CheckCircleOutline from "@mui/icons-material/CheckCircleOutline";
+import MultiStepHeader from "@bciers/components/form/components/MultiStepHeader";
+import { actionHandler } from "@bciers/actions";
+import { UUID } from "crypto";
+import FormBase from "@bciers/components/form/FormBase";
+import {
+  operationReviewSchema,
+  operationReviewUiSchema,
+} from "@reporting/src/app/utils/jsonSchema/operations";
+
+export default function OperationReview({
+  formData,
+  version_id,
+}: {
+  formData: any;
+  version_id: UUID;
+}) {
+  const customStepNames = [
+    "Operation Information",
+    "Facilities Information",
+    "Compliance Summary",
+    "Sign-off & Submit",
+  ];
```

Comment:
Nice! We'll probably refactor these into a common component for all the reporting stuff but for the first page 👌